meetings/agenda-minutes-2020-10-23.md

```diff
@@ -0,0 +1,96 @@
+# Oct 23, 2020
+
+## Attendees:
+- Ujjwal Sharma (USA)
+- Justin Grant (JGT)
+- Philip Chimento (PFC)
+- Richard Gibson (RGN)
+- Shane F Carr (SFC)
+
+## Agenda:
+
+### Finalize rounding modes [#1038](https://github.com/tc39/proposal-temporal/issues/1038)
+- PFC: Can we just make a note that the set of strings is provisional and is intended to match whatever Intl.NumberFormat does?
+- RGN: Agreed, that wouldn't hold up a review.
+- SFC: On the Intl.NumberFormat side JGT proposed an away-from-zero (anti-truncate) mode that is in ICU but not CSS.
+- USA: `'ceil'` would work for the lawyer billing case.
+- RGN: Was there a use case for this?
+- JGT: On the output of difference() you need it in the case where you're not sure of the sign but you want to round up.
+- RGN: That's true, but you can do `difference().abs().round({ roundingMode: 'ceil' })`.
+- JGT: That's three operations, whereas if you round in difference() it's only two.
+- [JGT/SFC/RGN met to follow up after the big meeting — summary is below]
+- RGN: Floating vs. non-floating distinction isn’t needed.
+- JGT: Add “away from zero”. Name = “expand” ?
+- RGN: Good, sounds like opposite of “trunc”.  We don’t want a name that sounds signed.
+- RGN: Rationale for choosing names:
+  1) The main four (ceil, floor, trunc, nearest) should be not confusable with each other.
+  2) We should derive the “half/tie-breaking” option names from the main four.
+- SFC: Let’s add all the tie-breaking options: the main four (ceil, floor, trunc, expand) and also even/odd. Names will be halfXxx, e.g. halfExpand.
+- SFC: camelCase for names?
+- JGT: OK. Temporal should defer to 402 on casing rules.
+- JGT: Is it OK that the default (halfExpand) is not obvious like the previous name “nearest”?
+- RGN: Yes, because the default will be the least likely name to type in. 
+- JGT: OK. I like the pattern of tiebreakprefixMode not modeTiebreakprefix because the former lumps all the tie-breaking modes together in auto-complete.
+- SFC: OK, we have consensus, I’ll write it up and sync with Intl.
+
+### `Rename `difference` to communicate directionality [#998](https://github.com/tc39/proposal-temporal/issues/998)
+- PFC: I think one of the options would be easier to implement, just renaming the method, and the other one would need to change the algorithm because the implicit relativeTo is `this`.
```

Comment:
Is there something wrong with the formatting of this section that's causing the entire section to be bold and blue?

Reply:
It was the extra backtick, thanks for fixing it @Ms2ger!